app/components/Home.js

```diff
@@ -174,14 +175,23 @@ export default class Home extends React.Component {
     }, 2000);
   };
 
+  onSearch = event => {
+    this.setState({ searchTerm: event.target.value })
+  }
+
   render() {
     const editEntry = this.state.editEntry || {};
+
+    this.state.entries.map(entry => {
```

Comment:
You shouldn't modify state directly. Instead use setState().

Reply:
good catch! :)